Options.py

```diff
@@ -1409,6 +1409,98 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
             link["item_pool"] = list(pool)
 
 
+class PlandoItem(typing.NamedTuple):
+    items: typing.Union[typing.List[str], typing.Dict[str, typing.Any]]
+    locations: typing.Optional[typing.List[str]]
+    world: typing.Union[int, str, bool, None, typing.Iterable[str], typing.Set[int]] = False
+    from_pool: bool = True
+    force: typing.Union[bool, typing.Literal["silent"]] = "silent"
+    count: typing.Union[int, bool, typing.Dict[str, int]] = False
```

Comment:
Since a `NamedTuple` is a `tuple`, and `tuple` has a `count` method, this is an invalid override.

Would a `@dataclass(frozen=True)` work instead of `NamedTuple`?